docs/fsharp/tutorials/getting-started/getting-started-visual-studio.md

```diff
@@ -14,35 +14,34 @@ ms.assetid: 8db75596-19a9-4eda-b20d-a12d517c8cc1
 
 # Getting started with F# in Visual Studio
 
-F# and the Visual F# tooling are supported in the Visual Studio IDE.  To begin, you should [download Visual Studio](https://www.visualstudio.com/downloads/download-visual-studio-vs), if you haven't already.  This article uses the Visual Studio 2015 Community Edition, but you can use F# with the version of your choice.
+F# and the Visual F# tooling are supported in the Visual Studio IDE.  To begin, you should [download Visual Studio](https://www.visualstudio.com/downloads/download-visual-studio-vs), if you haven't already.  This article uses the Visual Studio 2017 Community Edition, but you can use F# with the version of your choice.
 
-## Installing the Visual F# tools
+## Installing F# #
 
-Visual Studio will first initialize the installer.  After it is intilized, select **Custom** as shown here:
+If you're downloading Visual Studio for the first time, it will install the installer.  Install any version of Visual Studio 2017 from the installer.  If you already have it installed, click **Modify**.
```

Comment:
it will install the installer -> it will first install the Visual Studio Installer?